cmd/sops/main.go

```diff
@@ -443,6 +444,55 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:      "filestatus",
+			Usage:     "check the status of the file, returning encryption status",
+			ArgsUsage: `file`,
+			Flags:     []cli.Flag{},
+			Action: func(c *cli.Context) error {
+				// _, err := config.FindConfigFile(".")
+				// if err != nil {
+				// 	return common.NewExitError(err, codes.ErrorGeneric)
+				// }
+				if c.NArg() < 1 {
+					return common.NewExitError("Error: no file specified", codes.NoFileSpecified)
+				}
+
+				fileName := c.Args()[0]
+
+				inputStore := inputStore(c, fileName)
+				outputStore := outputStore(c, fileName)
+
+				svcs := keyservices(c)
+				opts := encryptOpts{
+					Cipher:            aes.NewCipher(),
+					InputStore:        inputStore,
+					OutputStore:       outputStore,
+					InputPath:         fileName,
+					UnencryptedSuffix: "",
+					EncryptedSuffix:   "",
+					EncryptedRegex:    "",
+					KeyGroups:         []sops.KeyGroup{},
+					GroupThreshold:    0,
+					KeyServices:       svcs,
+				}
+
+				fileBytes, err := ioutil.ReadFile(opts.InputPath)
+				if err != nil {
+					return common.NewExitError(fmt.Sprintf("Error reading file: %s", err), codes.CouldNotReadInputFile)
+				}
+				branches, err := opts.InputStore.LoadPlainFile(fileBytes)
+				if err != nil {
+					return common.NewExitError(fmt.Sprintf("Error unmarshalling file: %s", err), codes.CouldNotReadInputFile)
+				}
+				if err := ensureNoMetadata(opts, branches[0]); err != nil {
+					return common.NewExitError("File is encrypted", codes.FileAlreadyEncrypted)
```

Comment:
Hmmm, I'm wondering whether we should return an error when the file is not encrypted, or the other way around. What was your reasoning for choosing this over the alternative?

Reply:
Mainly due to the fact that there was an error code for "file is already encrypted" and not the other way round. So I would expect users to be already used to such an error.

IMO none is an error, and using JSON output like you requested in a comment above everything makes more sense and this error can probably be removed.

Reply:
Yeah, I agree. Let's make none of them be an error for now.